UI/js/common.js

```diff
@@ -0,0 +1,21 @@
+/*
+UI errors for user display
+1000 -> user errors
```

Comment:
Missing asteric for spacing

Reply:
This is just a comment and so we dont really need spaces here

Reply:
For consistency though it would be nice, as throughout the code this was how it was done

Reply:
normally we don't have spaces in comments since that would take up extra space, but sure